wordpress-mobile/AztecEditor-iOS
# Issue/603 add extra image attributes

Fixes #252 

To test:
 - Make sure the demo content is opening.
 - Make sure the unit tests are running correctly.
